docs/dev/setup/privacy-statement.rst

```diff
@@ -0,0 +1,260 @@
+Privacy
+^^^^^^^
+The privacy statement is stored in the ``./legal``  directory by default. You can override this value by setting the ``legal-path`` value in the ``application-artemis.yml``.
+The privacy statement is stored as markdown file. Currently, English and German as languages are supported.
+The privacy statement file has to follow the naming convention ``privacy_statement_<de|en>.md``.
+In case, you add only a file for one language, this file will always be shown regardless of the user's language setting.
+If you add a file for each language, the file will be shown depending on the user's language setting.
+
+In the following, the documentation provides a template in English and German for the privacy statement with placeholders that have to be replaced with the actual content.
+
+.. warning::
```

Comment:
Maybe personal preference:
I’d not put a complete template here since the privacy statement is very dependent on actual location, institution, etc. Instead, I’d give a short summary what kind of information is stored by Artemis by default, e.g.
- user name/email/matriculation number,
- anything the user submits to exercises/posts
- cookies only required ones, mostly login and locally saved UI theme preference
- the usual web server information as part of the log

Reply:
I personally think it's suitable given that the universities that use Artemis are almost entirely based in Germany, so I think quite some parts could be reused and it's just a template, so no need to use it in any form. For now, I would keep it.